sdk/src/main/java/com/podio/sdk/androidasynchttp/AndroidAsyncHttpRequest.java

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
The same goes for @author comments. That was needed before version control systems.
Now I can see who wrote any line of code and navigate the history. @author is just another annoying comment that I *have to* read before saying "ah, I dont care".